games-rpg/zsdx/zsdx-1.12.3.ebuild

```diff
@@ -38,12 +35,12 @@ src_configure() {
 
 src_install() {
 	cmake_src_install
-	newicon -s 48 build/icons/${PN}_icon_48.png ${PN}.png
-	newicon -s 256 build/icons/${PN}_icon_256.png ${PN}.png
+	newicon -s 48 "build/icons/${PN}_icon_48.png" "${PN}.png"
```

Comment:
I usually prefer to only quote things which I know may contain spaces, but this isn't wrong per se, it's just arguably "noisier". But it's a style thing.